crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -0,0 +1,214 @@
+/// See <https://starlederer.github.io/bloom/> for a visualization of the parametric curve
+/// used in Bevy as well as a visualization of the curve's respective scattering profile.
+#[allow(clippy::doc_markdown)]
+#[derive(Component, Reflect, Clone)]
+pub struct BloomSettings {
+    /// Controls the baseline of how much the image is scattered (default: 0.3).
+    ///
+    /// # In energy-conserving mode
+    /// The value represents how likely the light is to scatter.
+    ///
+    /// The value should be between 0.0 and 1.0 where:
+    /// * 0.0 means no bloom
+    /// * 1.0 means the light is scattered as much as possible
+    ///
+    /// # In additive mode
+    /// The value represents how much scattered light is added to
+    /// the image to create the glow effect.
+    ///
+    /// In this configuration:
+    /// * 0.0 means no bloom
+    /// * > 0.0 means a proportionate amount of scattered light is added
+    pub intensity: f32,
+
+    /// Low frequency contribution boost.
+    /// Controls how much more likely the light
+    /// is to scatter completely sideways (low frequency image).
+    ///
+    /// Comparable to a low shelf boost on an equalizer.
+    ///
+    /// # In energy-conserving mode
+    /// The value should be between 0.0 and 1.0 where:
+    /// * 0.0 means low frequency light uses base intensity for blend factor calculation
+    /// * 1.0 means low frequency light contributes at full power
+    ///
+    /// # In additive mode
+    /// The value represents how much scattered light is added to
+    /// the image to create the glow effect.
+    ///
+    /// In this configuration:
+    /// * 0.0 means no bloom
+    /// * > 0.0 means a proportionate amount of scattered light is added
+    pub low_frequency_boost: f32,
+
+    /// Low frequency contribution boost curve.
+    /// Controls the curvature of the blend factor function
+    /// making frequencies next to the lowest ones contribute more.
+    ///
+    /// Somewhat comparable to the Q factor of an equalizer node.
+    ///
+    /// Valid range:
+    /// * 0.0 - base base intensity and boosted intensity are linearly interpolated
+    /// * 1.0 - all frequencies below maximum are at boosted intensity level
+    pub low_frequency_boost_curvature: f32,
+
+    /// Tightens how much the light scatters (default: 1.0).
+    ///
+    /// Valid range:
+    /// * 0.0 - maximum scattering angle is 0 degrees (no scattering)
+    /// * 1.0 - maximum scattering angle is 90 degrees
+    pub high_pass_frequency: f32,
+
+    pub prefilter_settings: BloomPrefilterSettings,
+
+    /// Controls whether bloom textures
+    /// are blended between or added to each other. Useful
+    /// if image brightening is desired and a must-change
+    /// if prefilter_settings are used.
+    ///
+    /// # Recommendation
+    /// Set to Additive if prefilter_settings are
+    /// configured in a non-energy-conserving way,
+    /// otherwise set to EnergyConserving.
```

Comment:
Nitpick: Doc comment formatting

```suggestion
    /// Controls whether bloom textures
    /// are blended between or added to each other. Useful
    /// if image brightening is desired and a must-change
    /// if `prefilter_settings` are used.
    ///
    /// # Recommendation
    /// Set to [`BloomCompositeMode::Additive`] if `prefilter_settings` are
    /// configured in a non-energy-conserving way,
    /// otherwise set to [`BloomCompositeMode::EnergyConserving`].
```